### ISSUE TITLE
cannot access ‘chromedriver’: No such file or directory

### ISSUE THREAD

**Issue by mikemadison13** [acquia/blt#2738]
My system information:
* Operating system type: Travis CI / Linux
* BLT version: 9.x-dev #9fb6893

When I run this command:
```
${BLT_DIR}/scripts/linux/install-chrome.sh ${COMPOSER_BIN}
```

I get the following output:
```
# Installs chromedriver to vendor/bin.
${BLT_DIR}/scripts/linux/install-chrome.sh ${COMPOSER_BIN}
--2018-04-17 21:06:40--  http://chromedriver.storage.googleapis.com/2.38/chromedriver_linux64.zip
Resolving chromedriver.storage.googleapis.com (chromedriver.storage.googleapis.com)... 209.85.200.128, 2607:f8b0:4001:c14::80
Connecting to chromedriver.storage.googleapis.com (chromedriver.storage.googleapis.com)|209.85.200.128|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-04-17 21:06:40 ERROR 404: Not Found.
unzip:  cannot find or open chromedriver_linux64.zip, chromedriver_linux64.zip.zip or chromedriver_linux64.zip.ZIP.
chmod: cannot access ‘chromedriver’: No such file or directory
mv: cannot stat ‘chromedriver’: No such file or directory
rm: cannot remove ‘chromedriver_linux64.zip’: No such file or directory
```

And I expected this to happen:
It looks like the chrome drive we are trying to download is not available online currently see -- http://chromedriver.storage.googleapis.com/2.38/chromedriver_linux64.zip which outputs to 

```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Details>
No such object: chromedriver/2.38/chromedriver_linux64.zip
</Details>
</Error>
```

**Comment by mikemadison13** [comment]
it looks like http://chromedriver.storage.googleapis.com/LATEST_RELEASE reports that 2.3.8 is the latest release, but they haven't actually deployed any files yet.

**Comment by mikemadison13** [comment]
this can be temporarily fixed by patching manually with PR #2739 which hard codes the version to 2.37 until the 2.38 release is completed.

**Comment by mikemadison13** [comment]
this should no longer be an issue, as the 2.38 url above now resolves.
